packages/perseus/src/widgets/radio/radio-component.tsx

```diff
@@ -268,12 +268,12 @@ class Radio extends React.Component<Props> implements Widget {
      */
     showRationalesForCurrentlySelectedChoices: (
```

Comment:
My only reservation with these changes are touching some of the things we need to address in the future: `showRationalesForCurrentlySelectedChoices`, `reviewModeRubric`, etc. I think it's fine to ship the changes, there's just something in the back of my mind worried that we'll lose some context by renaming things around these APIs before we actually work on them.

I have no justification for this concern, so no action needed.

Reply:
@handeyeco I can go through and leave a link to this PR on the relevant tickets and just point out that the naming may have changed and here's where you can see where that happened. I think I might also suggest that if it hasn't been renamed that it get renamed in the course of whatever ticket work happens. Does that sound like a good way to help prevent some loss of context? 

I am also 100% okay with putting out another PR reverting any naming changes if we decide after I've landed this that we don't actually want some of these things renamed.